doc/classes/String.xml

```diff
@@ -751,7 +751,7 @@
 			<argument index="1" name="len" type="int">
```

Comment:
In the end there should be "default="-1"", so it explicitly shows it's optional.

Reply:
If defined properly, `godot --doctool .` should add it.

Reply:
What akien said
![image](https://user-images.githubusercontent.com/2223172/57152988-6915aa80-6dd5-11e9-9561-371a12094e5e.png)

Reply:
It would be good to amend the commit to include this though.

Reply:
How to add it automatically to xml? Because it only shows for me in the build.

Reply:
I gave the command above.

Reply:
Doing this on Windows opens weirdly upscaled project manager :/ (at least for me)
I changed it manually.